pkgs/by-name/ba/babl/package.nix

```diff
@@ -31,6 +31,17 @@ stdenv.mkDerivation (finalAttrs: {
     ./dev-prefix.patch
   ];
 
+  postPatch = ''
```

Comment:
Could you please move it next to other phases like in gegl? For GNOME-adjacent packages, we prefer having all the phases in the bottom, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.